vllm/entrypoints/openai/serving_completion.py

```diff
@@ -280,6 +281,20 @@ async def create_completion(
 
             final_res_batch_checked = cast(list[RequestOutput], final_res_batch)
 
+            for final_res in final_res_batch_checked:
+                for output in final_res.outputs:
+                    if output.finish_reason == "error":
+                        logger.error(
+                            "Request-level error for request %s: %s",
+                            request_id,
+                            output.stop_reason or "unknown",
+                        )
```

Comment:
Might be better to omit the logging here given we already log where the error occurs?